### DISCUSSION TITLE
Are many scenes empty in the SUNRGB-D dataset

### DISCUSSION THREAD

**Discussion by Agito555** [open-mmlab/mmdetection3d#2899] [General]
I attempted to print the count of ground truth labels for each scene in the SUNRGB-D training set and found that many scenes are empty. I'm curious to know if this is a characteristic of the dataset or if there's an issue with my data processing that's resulting in incorrect data.
